### DISCUSSION TITLE
blog/how-to-translate-a-book/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#307] [Announcements]
# blog/how-to-translate-a-book/

如何翻译一本外文书，从图书引进到翻译出版全攻略。

https://jimmysong.io/blog/how-to-translate-a-book/

**Comment by Haiming-Yang** [comment]
什么时候联系原作者呢？是出版社代为联系吗？

**Reply by rootsongjc** [reply]
一般让出版社去联系原作出版社
